es-app/src/Gamelist.cpp

```diff
@@ -142,7 +152,7 @@ void parseGamelist(SystemData* system)
 			else if(!file->isArcadeAsset())
 			{
 				std::string defaultName = file->metadata.get("name");
-				file->metadata = MetaDataList::createFromXML(GAME_METADATA, fileNode, relativeTo);
+				file->metadata = MetaDataList::createFromXML(i == 0 ? GAME_METADATA : FOLDER_METADATA, fileNode, relativeTo);
```

Comment:
This flag got mandatory to differentiate between recognized subelements of <game/> and <folder/>